app/_atoms/Checkbox.tsx

```diff
@@ -1,13 +1,22 @@
-import { CheckboxProps } from "../types"
+import { CheckboxProps } from "../types";
 
-const Checkbox: React.FC<CheckboxProps> = ({label,value,onChange,checked},name) => {
+const Checkbox: React.FC<CheckboxProps> = (
+  { label, value, onChange, checked },
+  name
```

Comment:
hocam buradaki name'in maksadini anlayamadim, destructuring yapacaksaniz süslü parantezin icinde olmali. ayrica label'i destructuring yaptiginiz icin eger input icine props olarak gönderecekseniz yani 'name={value}' yapacaksaniz üst tarafta yazmaniza gerek yok. yani söyle olmali:    const Checkbox: React.FC<CheckboxProps> = ({ label, value, onChange, checked }) => {
  return (
    <div className="flex gap-2 items-center">
      <input
        type="checkbox"
        name={value}
        value={value}
        onChange={onChange}
        checked={checked}
        className="h-6 w-6 text-secondary border-2 border-[#0275B4] focus:ring-2 focus:ring-[#0275B4]"
      />
      <span className="text-[#0275B4] text-[14px]">{label}</span>
    </div>
  );
};

Reply:
Hocam bu kismi ben yazmamistim ama rebase true  yaptigim  icin sanki ben yazmisim gibi olmus. Bu bolum atomda yapilmis, bende hic degisiklik yapmadim. Acikcasi bende bu kismin neden yapildigini bilmiyorum.

Reply:
o zaman kalsin öyle